xoa/resource_xenorchestra_vm.go

```diff
@@ -230,15 +230,6 @@ func resourceVmSchema() map[string]*schema.Schema {
 						Type:     schema.TypeString,
 						Optional: true,
 						Computed: true,
-						StateFunc: func(val interface{}) string {
-							unformattedMac := val.(string)
-							mac, err := net.ParseMAC(unformattedMac)
-							if err != nil {
-								panic(fmt.Sprintf("Mac address `%s` was not parsable. This should never happened because Terraform's validation should happen before this is stored into state", unformattedMac))
-							}
-							return mac.String()
-
-						},
```

Comment:
@ravager-dk what was the rationale behind removing this? Doing so breaks the TestAccXenorchestraVm_createAndUpdateWithMacAddress [acceptance test](https://github.com/terra-farm/terraform-provider-xenorchestra/blob/3e5215187f7ed75394b5d9fe6c5fbfb6f8646b65/xoa/resource_xenorchestra_vm_test.go#L674).

Reply:
This got added into the PR by mistake from my other PR #233. Should I redo this PR completely without it, or can you edit this out?

Reply:
You don't need to redo the PR, but you do need to revert the change on the `empty-mac` branch and push to your remote again.

I can make the change if you want, but that would require opening a new PR (since I don't have commit access to your remote). I'm happy to do that if you prefer. If we go that route, we can make sure to preserve your commits so your contributions are still counted.

Reply:
I will revert the the change and push it anew. :)